api/v1beta1/operatorpolicy_types.go

```diff
@@ -143,6 +152,12 @@ type OperatorPolicySpec struct {
 	// approval is not affected by this setting. This setting has no effect when the policy is in
 	// 'mustnothave' mode. Allowed values are "None" or "Automatic".
 	UpgradeApproval string `json:"upgradeApproval"`
+
+	//+kubebuilder:default={}
+	// ComplianceConfig defines how resource statuses affect the OperatorPolicy status and compliance.
+	// When set to Compliant, the condition does not impact the OperatorPolicy compliance.
+	// When set to NonCompliant, the condition cause the OperatorPolicy to become NonCompliant.
```

Comment:
```suggestion
	// When set to NonCompliant, the condition causew the OperatorPolicy to become NonCompliant.
```

Reply:
Can I assume you meant "causes"?

Reply:
Yup, you caught it before the edit :laughing: